.github/workflows/rust.yml

```diff
@@ -21,3 +19,17 @@ jobs:
       run: cargo build --verbose
     - name: Run tests
       run: cargo test --verbose
+
+  build-android:
+    name: Build for ${{ matrix.target }}
+    runs-on: ubuntu-latest
+    container:
+      image: devduttshenoi/android-builder
+    strategy:
+      fail-fast: false
+      matrix:
+        target: [aarch64-linux-android, armv7-linux-androideabi, x86_64-linux-android, i686-linux-android]
```

Comment:
Do we need all targets? what is the reason it has been included at PR stage?

Reply:
@de-sh With this ci will error if compile fails for android

Reply:
@de-sh I see no reason not to build for all supported targets